database/setup.js

```diff
@@ -5,7 +5,7 @@ const seed = require('./seed')
 MongoClient
   .connect(process.env.MONGODB_URI, { useNewUrlParser: true })
   .then(client => {
-    const db = client.db('computer-parts-app')
+    const db = client.db('heroku_skz5tpxx')
```

Comment:
ouch. You need to delete this and re-provision a new database. The database name should be added to the name of your `MONGODB_URI` environment variable. Then call `client.db()` with no arguments in your server and setup.

```
MONGODB_URI=mongodb://host:port/db-name
```